merge_rules.py

```diff
@@ -193,13 +198,22 @@ def parse_command_line(program_info):
 
     parser.add_argument(
         '--replace_alphas_only',
-        help = 'Only merge the alpha structures from input. Keeping everything of generic rule set as is.',
+        help = 'Only use the alphas from the input rule set. Keeping everything else of generic rule set as is.',
         required = False,
         action='store_const',
         const= not program_info['replace_alphas_only'],
         default = program_info['replace_alphas_only']
     )
 
+    parser.add_argument(
+        '--merge_base_only',
+        help = 'Only merge the base structures from input. Keeping grammars and captalization as is.',
```

Comment:
By using this feature we will notice soon enough if this is a good name/description :)